content/en/cosign/openid_signing.md

```diff
@@ -4,71 +4,34 @@ category: "Cosign"
-Generating ephemeral keys...
-Retrieving signed certificate...
-Your browser will now be opened to:
-https://accounts.google.com/o/oauth2/v2/auth?access_type=offline&client_id=237800849078-rmntmr1b2tcu20kpid66q5dbh1vdt7aj.apps.googleusercontent.com&redirect_uri=http%3A%2F%2F127.0.0.1%3A5556%2Fauth%2Fgoogle%2Fcallback&response_type=code&scope=openid+email&state=8slZXeZhwKQofg%3D%3D
-Pushing signature to: gcr.io/user-vmtest2/demo:sha256-97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36.sig
-```
-
-Keyless verifying:
-
-```shell
-$ COSIGN_EXPERIMENTAL=1 cosign verify gcr.io/user-vmtest2/demo
-The following checks were performed on all of these signatures:
-  - The cosign claims were validated
-  - The claims were present in the transparency log
-  - The signatures were integrated into the transparency log when the certificate was valid
-  - Any certificates were verified against the Fulcio roots.
-Certificate subject:  user@google.com
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36"},"Type":"cosign container image signature"},"Optional":null}
-```
-
-That's it! No keys!
-The rest of the flags (annotations, claims, tlog, etc.) should all work the same.
-
-## Overview
-
-This uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
+Keyless signing uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
 
 Information on the Fulcio root CA can be found in the [fulcio repository](https://github.com/sigstore/fulcio).
 
-### Keys
+## Keys
 
-The root CA keys are hard-coded in Cosign today. They can only be changed by recompiling the binary. This will be made more configurable in the future.
+The root CA keys are hard-coded in Cosign today. They can only be changed by recompiling the binary. 
 
-### OAuth flows
+## OAuth flows
 
 Cosign supports two OAuth flows today: the standard flow and the device flow.
 
-When there is no terminal attached (non-interactive mode), `cosign` will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
+When there is no terminal attached (non-interactive mode), Cosign will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
 
-### Identity tokens
+## Identity tokens
 
 In automated environments, cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` field must contain `sigstore`.
```

Comment:
You can also mention the environment variable `SIGSTORE_ID_TOKEN` as an alternative